packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,364 @@
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">The Effects API</h2>
+            <p>All Effects are reactive components, so all properties can be changed during runtime. 
+            This example demonstrates the BlendMode API, which allows you to blend effects like you would
+            layers in Photoshop. We use a &lt;color-grade-effect&gt;, which allows you to alter the look of your model.
+            </p>
+          </div>
+          <example-snippet stamp-to="effects" highlight-as="html">
+            <template>
+<model-viewer id="blendViewer" camera-controls touch-action="pan-y" ar src="../../shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut">
+  <effect-composer>
+    <color-grade-effect id="color-grade" contrast="0.5" saturation="0.4"></color-grade-effect>
+  </effect-composer>
+  <div class="controls">
+    <label for="opacity">Opacity</label>
+    <input id="opacity" type="range" min="0" max="1" step="0.01" value="1">
+    <label for="blend-mode">Blend Mode:</label>
+    <select id="blend-mode">
+        <option value="default">Default</option>
+        <option value="skip">Skip</option>
+        <option value="add">Add</option>
+        <option value="subtract">Subtract</option>
+        <option value="divide">Divide</option>
+        <option value="negation">Negation</option>
+    </select>
+  </div>
+</model-viewer>
+<script type="module">
+const blendViewer = document.querySelector("model-viewer#blendViewer");
+const colorGradeEffect = blendViewer.querySelector('#color-grade');
+const opacity = blendViewer.querySelector('#opacity');
+const mode = blendViewer.querySelector('#blend-mode');
+
+opacity.addEventListener('input', (e) => colorGradeEffect.opacity = e.target.value);
+mode.addEventListener('change', (e) => colorGradeEffect.blendMode = e.target.value);
+</script>
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="sample">
+      <div id="selective-effects" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Selective Effects</h2>
+            <p>Some of the implemented effects are selective and may be applied to only certain meshes in your model. 
+            Currently, this is exclusive to &lt;selective-bloom-effect&gt; and &lt;outline-effect&gt;. The default behavior for selective
+            effects is to use the default <code>selection</code> provided by the &lt;effect-composer&gt;, which contains a set of
+            all meshes in the model.<br/>
+            By providing an array of mesh names to the <code>selection</code> attribute, you may enable the effects only for those meshes.<br/>
+            This example demonstrates the application of the Bloom effect exclusively to Meshes with an emissive material:</p>
```

Comment:
Are all the materials emissive? It doesn't look very selective...
<img width="592" alt="image" src="https://user-images.githubusercontent.com/1649964/229639858-b5ba248d-f32e-42ba-b7bb-d4063b2316ca.png">

Reply:
![image](https://user-images.githubusercontent.com/54665621/229706117-c63755aa-6946-4cba-b932-68138bff6ff1.png)

Not getting this issue either in firefox, managed to reproduce using chrome. I'll look into it, the selection also works weirdly for outline so its probably a bug on my end.

Reply:
Interesting bug, turns out that despite the documentation that lit-element array properties update only on changing the reference https://lit.dev/docs/components/properties/#mutating-properties, the above example where I simply pushed values into the array triggered the update loop on firefox on each push. On chrome this was not the case.